content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -1,124 +1,28 @@
-and [docs](https://github.com/kubermatic/kubeone/tree/master/docs) for details on how to use it.
-
-Please take note of the [recommended hardware and networking requirements](../../requirements/cluster_requirements/)
-before provisioning a cluster.
-
-### 2. Install Kubermatic Dependencies
-
-When using Helm 2, install Tiller into the seed cluster first:
-
-```bash
-kubectl create namespace kubermatic
-kubectl create serviceaccount -n kubermatic tiller
-kubectl create clusterrolebinding tiller-cluster-role --clusterrole=cluster-admin --serviceaccount=kubermatic:tiller
-helm --service-account tiller --tiller-namespace kubermatic init
-```
-
-#### NodePort Proxy
-
-Kubermatic requires the NodePort Proxy to be installed in each seed cluster. The proxy is shipped as a
-[Helm](https://helm.sh) chart in the kubermatic-installer repository.
-
-As the NodePort Proxy Docker image is in a private registry, you need to configure the Docker Pull Secret for
-the Helm chart. You can re-use the `values.yaml` used during the installation or create new one and configure it like
-so:
-
-```yaml
-kubermaticOperator:
-  # insert the Docker authentication JSON provided by Loodse here
-  imagePullSecret: |
-    {
-      "auths": {
-        "quay.io": {....}
-      }
-    }
-```
-
-#### Cluster Backups
-
-Kubermatic performs regular backups of user cluster by snapshotting the etcd of each cluster. By default these backups
-are stored locally inside the cluster, but they can be reconfigured to work with any S3-compatible storage.
-The in-cluster storage is provided by [Minio](https://min.io/) and the accompanying `minio` Helm chart.
-
-If your cluster has no default storage class, it's required to configure a class explicitely for Minio. You can check
-the cluster's storage classes via:
-
-```bash
-kubectl get storageclasses
-#NAME                 PROVISIONER            AGE
-#kubermatic-fast      kubernetes.io/gce-pd   195d
-#standard (default)   kubernetes.io/gce-pd   2y43d
-```
-
-{{% notice note %}}
-Minio does not use `kubermatic-fast` because it does not require SSD speeds. A larger HDD is preferred.
-{{% /notice %}}
-
-To configure the storage class and size, extend your `values.yaml` like so:
-
-```yaml
-minio:
-  storeSize: '200Gi'
-  storageClass: hdd
-```
-
-It's also advisable to install the `s3-exporter` Helm chart, as it provides basic metrics about user cluster backups.
-
-#### Install Charts
-
-With this you can install the chart:
-
-```bash
-cd kubermatic-installer
-helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace nodeport-proxy nodeport-proxy charts/nodeport-proxy/
-helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace minio minio charts/minio/
-helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace s3-exporter s3-exporter charts/s3-exporter/
-```
```

Comment:
We still must tell users to install these components, even for the CE version in the same cluster as the master. The "Install Kubermatic" page ONLY handles the master, so it does not mention these three Helm charts.

Reply:
added